saber/boss_access/README.md

```diff
@@ -96,30 +72,18 @@ Type: int
 
 Description: Bounds for volume.
 
-##### padding
-
-Type: int
-
-Description: How much padding to use. Padding is applied to pulled images and stripped from pushed images.
-
 ##### Output
 
 Type: File
 
 Description: Output filename
 
-##### s3-only
-
-Type: Flag
-
-Description: If set, no local file will be saved and the resulting cutout will be uploaded to S3. 
-
 ### Pushing data
 
 #### Usage
 
-```
-boss_access.py push (-c CONFIG | -t TOKEN) [-b BUCKET] --coll COLL --exp EXP --chan CHAN --coord COORD --dtype DTYPE --itype ITYPE [--res RES] [--xmin XMIN][--xmax XMAX] [--ymin YMIN] [--ymax YMAX] [--zmin ZMIN] [--zmax ZMAX] [--padding PADDING] -o OUTPUT [--s3-only]
+```bash
+boss_access.py push (-c CONFIG | -t TOKEN) --coll COLL --exp EXP --chan CHAN [--res RES] [--xmin XMIN][--xmax XMAX] [--ymin YMIN] [--ymax YMAX] [--zmin ZMIN] [--zmax ZMAX] --type TYPE -i INPUT
```

Comment:
We should add documentation that says what bossdb stack it defaults to. I'm assuming if you provide a token it defaults to 'api.bossdb.io' but maybe if you want to use 'api.theboss.io' you have to provide a config file with the host/protocol you want.

Reply:
I'm unfamiliar with api.theboss.io, is there also a public token for that host?

Reply:
Yeah so thats the MICrONS stack so basically there are a couple different "bosses" out there. There is the public stack (api.bossdb.io), microns stack (api.theboss.io), test stack (api.bossdbtest.io), and personal dev stacks (api-xenesd1.thebossdev.io). All of them have separate data, user accounts, settings, etc. So to use this tool for another stack there should be two additional inputs:

`protocol` (optional, str): defaults to `'https'` 
`host` (optional, str): defaults to `'api.bossdb.io'`

That way for regular use-cases I dont have to worry about remembering the public boss api domain but still have the option to use another stack if needed.